src/topics/doc/security.md

```diff
@@ -44,7 +44,9 @@ Fine-grained write access control within a single domain can be achieved using _
 > 🚧 Using constraints for fine-grained write access control is currently experimental. You can read the [white paper here](https://github.com/m-ld/m-ld-security-spec/blob/main/design/suac.md), and explore the [prototype support](https://js.m-ld.org/classes/writepermitted.html) in the Javascript engine. Please [contact us](/hello/) to discuss your security requirements.
 
 ### Auditing & Non-Repudiation
-Once a user is authorised to the application, it may be important to record their activity, as well as that of any other system actor such as a bot, in tamper-proof way, for later auditing. This can generally be achieved with the use of audit stamps (time & user) on updates. If necessary, these stamps can be digitally signed by the app.
+Once a user is authorised to the application, it may be important to record their activity, as well as that of any other system actor such as a bot, in tamper-proof way, for later auditing. In common with other systems, it is usually most efficient to use a dedicated system component for this. In an app using **m-ld**, a clone of the data can be located with the audit logging component.
```

Comment:
Spotted one further (albeit minor) omission: the phrase "in tamper-proof way" should read "in *_a_* tamper-proof way".